components/k8s-model-server/http-proxy/Makefile

```diff
@@ -1,16 +1,45 @@
+# Copyright 2017 Google Inc.
```

Comment:
2018?

Reply:
ahh, good catch. I noticed this file didn't have a header so I copied it over from another file without changing the year